mark_line_on_image.py

```diff
@@ -0,0 +1,54 @@
+import argparse
+import cv2
+from find_line_in_image import find_line_contour
+

```

Comment:
You are using a global parameter named image, but it is not declared, to begin with.
And in function 'mark_rectangle_contour' it is not declared as global but still being used in the function.

Reply:
it is declared in the parser and the mark_rectangle_contour the image shown is a copy of the global image